pkgs/render/render.go

```diff
@@ -71,8 +69,8 @@ func applyFromBinData(logger logr.Logger, filePath string, data map[string]strin
 	if err != nil {
 		return err
 	}
-	if cfg != nil {
-		if err := ctrl.SetControllerReference(cfg, obj, scheme); err != nil {
+	if owner != nil {
+		if err := ctrl.SetControllerReference(owner, obj, client.Scheme()); err != nil {
```

Comment:
It is also unclear why removing the scheme input variable to using client.Scheme() was done. The commit message does not explain this.